package/python-pyqt5/0001-fix-configure.patch

```diff
@@ -0,0 +1,74 @@
+diff -purN PyQt5-5.14.2.orig/configure.py PyQt5-5.14.2/configure.py
+--- PyQt5-5.14.2.orig/configure.py	2020-04-02 07:58:06.029642000 -0700
++++ PyQt5-5.14.2/configure.py	2020-06-12 13:57:52.786605257 -0700
+@@ -1,19 +1,19 @@
+ # This script generates the Makefiles for building PyQt5.
+ #
+ # Copyright (c) 2020 Riverbank Computing Limited <info@riverbankcomputing.com>
+-# 
++#
+ # This file is part of PyQt5.
+-# 
++#
+ # This file may be used under the terms of the GNU General Public License
+ # version 3.0 as published by the Free Software Foundation and appearing in
+ # the file LICENSE included in the packaging of this file.  Please review the
+ # following information to ensure the GNU General Public License version 3.0
+ # requirements will be met: http://www.gnu.org/copyleft/gpl.html.
+-# 
++#
+ # If you do not wish to use this file under the terms of the GPL version 3.0
+ # then you may purchase a commercial license.  For more information contact
+ # info@riverbankcomputing.com.
+-# 
++#
+ # This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+ # WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+ 
+@@ -627,7 +627,7 @@ class TargetConfiguration:
+                 self.pyqt_sip_dir)
+         self.pyuic_interpreter = parser.get(section, 'pyuic_interpreter',
+                 self.pyuic_interpreter)
+- 
++
+     def from_introspection(self, verbose, debug):
+         """ Initialise the configuration by introspecting the system. """
+ 
+@@ -709,7 +709,8 @@ class TargetConfiguration:
+         if test is None:
+             error("Failed to determine the detail of your Qt installation. Try again using the --verbose flag to see more detail about the problem.")
+ 
+-        lines = run_test_program('QtCore', test, verbose)
++        #lines = run_test_program('QtCore', test, verbose)
```

Comment:
Your commit message explained why you are skipping the test; a similar comment here would be good here too.

Reply:
updated with a simple comment.